### PR TITLE
Errata print01-ch02-compilation-and-dispatch-find

### PR DESCRIPTION
**Pull request by weigangd**
Small error in chapter 02. 

Love the book so far!

### CODE REVIEW

**Review by jonhoo — CHANGES_REQUESTED** [review]
Ah, good catch! You should only need single backticks around code words though — would you mind updating those?

**Comment by weigangd** [comment]
Done

File: _errata/print01-ch02-compilation-and-dispatch-find.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+chapter: 2
+page: 27
+kind: typo
+reporter: weigangd
+date: 2022-01-12
+# fixed: 2021-01-01
+---
+In the last paragraph "[...] all the compiler can do for `find` in Listing 2-2 [...]":
+Listing 2-2 does not have a `find` function. It should probably say "[...] all the compiler can do for `contains` in Listing 2-2 [...]" .
```
Comment (jonhoo) [inline _errata/print01-ch02-compilation-and-dispatch-find.md]:
```suggestion
Listing 2-2 does not have a `find` function. It should say "[...] all the compiler can do for `contains` in Listing 2-2 [...]" .
```

**Review by jonhoo — APPROVED** [review]

**Comment by jonhoo** [comment]
Thank you!

**Pull request merged**